renderdoc/data/glsl/glsl_globals.h

```diff
@@ -167,7 +167,8 @@ vec3 CalcCubeCoord(vec2 uv, int face)
 #define RESTYPE_TEXBUFFER 0x9
 #define RESTYPE_TEX2DMS 0xA
 #define RESTYPE_TEX2DMSARRAY 0xB
-#define RESTYPE_TEXTYPEMAX 0xB
+#define RESTYPE_TEXEXTERNAL 0xC
```

Comment:
Adding a new resource type here doesn't make sense without at least corresponding shader changes. Otherwise setting a new texture slot in code will refer to a non-existing texture binding and cause textures to not display correctly. As above when talking about `TextureType`, external textures should not be treated as a different type of texture and instead should be accessed via `RESTYPE_TEX1D`, `RESTYPE_TEX2D`, `RESTYPE_TEX2DMS` etc depending on their actual type.